examples/experimental/local-image/serveLocalGvmi.ts

```diff
@@ -0,0 +1,40 @@
+import { GolemNetwork, serveLocalGvmi } from "@golem-sdk/golem-js/experimental";
+import { fileURLToPath } from "url";
+
+// get the absolute path to the local image in case this file is run from a different directory
+const localImagePath = fileURLToPath(new URL("alpine.gvmi", import.meta.url).toString());
+
+const server = serveLocalGvmi(localImagePath);
+const golem = new GolemNetwork({
+  yagna: {
+    apiKey: "try_golem",
+  },
+  package: {
+    localImageServer: server,
+  },
+});
+
+async function main() {
+  console.log("Serving local image to the providers...");
+  await server.serve();
+  await golem.init();
+
+  const job = golem.createJob<string>();
```

Comment:
Not a fan of using the `experimental` APiI here, but I know where you're coming from :)

Reply:
I think we can remove this example and add one in `task-executor` repo once the feature is released